### PR TITLE
Remove delete buttons: Accounts, Avatars, Scenes

### PR DESCRIPTION
**Pull request by robin-k-wilson**
Removes unimplemented delete button as an option for editing accounts, scenes, avatars.
Fixed some errors in console about styling.

### CODE REVIEW

File: admin/src/react-components/toolbar-without-delete.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Toolbar, SaveButton } from "react-admin";
+
+export const ToolbarWithoutDelete = props => {
```
Comment (netpro2k) [inline admin/src/react-components/toolbar-without-delete.js:4]: Clearly we should have done this a long time ago heh, nice and simple solution. I might have called this `SaveButtonToolbar` or `SaveOnlyToolbar`, but that's personal preference.

File: admin/src/react-components/accounts.js [hunk 2]
```diff
@@ -257,11 +258,11 @@ export const AccountList = withStyles(styles)(
 );
 
 export const AccountEdit = withStyles(styles)(props => {
-  const { classes } = props;
+  const { classes, ...other } = props;
 
   return (
-    <Edit {...props}>
-      <SimpleForm>
+    <Edit {...other}>
```
Comment (netpro2k) [inline admin/src/react-components/accounts.js:264]: Was passing `classes` through breaking something?

File: admin/src/react-components/accounts.js [hunk 3]
```diff
@@ -168,10 +169,10 @@ export const AccountList = withStyles(styles)(
         }, 6000);
       }
       render() {
-        const { classes } = this.props;
+        const { classes, refreshView, ...other } = this.props;
```
Comment (netpro2k) [inline admin/src/react-components/accounts.js:172]: we are no longer passing through `refreshView` here. Is that intended?

File: admin/src/react-components/accounts.js [hunk 4]
```diff
@@ -168,10 +169,10 @@ export const AccountList = withStyles(styles)(
...
         return (
           <>
-            <Card classes={{ root: classes.searchCard }}>
+            <Card className={classes.searchCard}>
```
Comment (netpro2k) [inline admin/src/react-components/accounts.js:175]: Hmm why does this change from `classes` to `className`? Quick search shows that some react-admin stuff expects `classes`, not sure if this is one of them. Possible it was just wrong before. Did you notice it breaking something?

File: admin/src/admin.js [hunk 5]
```diff
@@ -214,7 +214,10 @@ const HiddenAppBar = withStyles({
       display: "none"
     }
   }
-})(props => <AppBar {...props} className={props.classes.hideOnDesktop} />);
+})(props => {
+  const { classes, ...other } = props;
+  return <AppBar {...other} className={classes.hideOnDesktop} />;
```
Comment (netpro2k) [inline admin/src/admin.js:219]: Why do we no longer want to pass classes down?

File: admin/src/react-components/accounts.js (hunk 2 above)
Comment (robin-k-wilson) [reply admin/src/react-components/accounts.js:264]: Yes, was throwing an error in my console. Material ui was not liking it.

File: admin/src/react-components/accounts.js (hunk 3 above)
Comment (robin-k-wilson) [reply admin/src/react-components/accounts.js:172]: Yes, thanks for checking, it's being used where it needs to be in the function above outside of render().

File: admin/src/react-components/accounts.js (hunk 2 above)
Comment (netpro2k) [reply admin/src/react-components/accounts.js:264]: Ah right, you mentioned that "Fixed some errors in console about styling".

File: admin/src/react-components/accounts.js (hunk 3 above)
Comment (robin-k-wilson) [reply admin/src/react-components/accounts.js:174]: Added comment to make this clearer: // Do not pass refreshView to other components, only onCreateAccounts()

Was throwing an error getting passed in.

File: admin/src/react-components/accounts.js (hunk 4 above)
Comment (robin-k-wilson) [reply admin/src/react-components/accounts.js:177]: Double checked, both ways work. I've worked with material UI before and this is the simpler preferred way - no errors in console, style is applied!

File: admin/src/react-components/toolbar-without-delete.js (hunk 1 above)
Comment (robin-k-wilson) [reply admin/src/react-components/toolbar-without-delete.js:4]: Open to changing it -- I don't feel strongly -- though I think the name is clear for its purpose, I'd add a comment if I changed it. When we integrate delete functionality we only need to search for "Delete" where it's referenced.

File: admin/src/admin.js (hunk 5 above)
Comment (robin-k-wilson) [reply admin/src/admin.js:219]: Throwing the error in console from Material UI, styles still work as intended with the new syntax.

File: admin/src/react-components/accounts.js (hunk 3 above)
Comment (robin-k-wilson) [reply admin/src/react-components/accounts.js:174]: Added eslint ignore line as well.

File: admin/src/react-components/toolbar-without-delete.js (hunk 1 above)
Comment (netpro2k) [reply admin/src/react-components/toolbar-without-delete.js:4]: Yep I don't have strong preference either, just wanted to note.

**Pull request merged**